gtnet/run.py

```diff
@@ -1,38 +1,56 @@
 from .model import load_model
-from .sequence import _get_DNA_map, get_sequences, get_bidir_seq
-from .utils import get_species_pred, get_label_file
+from .sequence import _get_DNA_map, batch_sequence
+from .utils import get_taxon_pred, get_label_file
 from .utils import get_logger, get_data_path
+import ruamel.yaml as yaml
 import numpy as np
 import pandas as pd
 import argparse
+import skbio
+from skbio.sequence import DNA
+import json
 import sys
+import os
 
 
-def get_predictions(fasta_path, domain, vocab, output_dest=None, **kwargs):
+def get_predictions(fasta_path, output_dest=None, **kwargs):
     if fasta_path is None:
-        logging.error('Please provide a fasta path!')
+        logging.error('The path provided is not a fasta file')
         exit()
 
-    model = load_model(domain=domain)
+    model = load_model()
+
     input_name = model.get_inputs()[0].name
-    chars, basemap, rcmap = _get_DNA_map()
+    chars, basemap = _get_DNA_map()
+
+    deploy_path = 'gtnet/gtnet.deploy' #not ideal
+    config_path = os.path.join(deploy_path, 'config.yml')
+    with open(os.path.join(deploy_path, 'manifest.json'), 'r') as f:
+        manifest = json.load(f)
```

Comment:
This isn't getting used anywhere, but it should be. I'd recommend using this to get the model paths also.